docs/dev.rst

```diff
@@ -204,132 +204,30 @@ corresponding user in the oidcprovider service like this:
 
    docker-compose exec oidcprovider /code/manage.py createuser yourname yourpassword yourname@example.com
 
-Running ``gunicorn`` locally
```

Comment:
I have no idea if any of this works because I've never used it. I didn't want to go through it and update it. I figure if I haven't used it since I started maintaining Tecken, it's worth removing.